src/ir/datalog/program.h

```diff
@@ -92,6 +92,11 @@ class ArgumentType {
 
   std::string DebugPrint() const { return absl::StrCat(kind_, name_); }
 
+  bool operator==(const ArgumentType& otherType) const {
```

Comment:
Refer to the fields directly from `otherType`.

Reply:
fixed